nipype/interfaces/afni/model.py

```diff
@@ -153,7 +153,7 @@ class DeconvolveInputSpec(AFNICommandInputSpec):
         'instead of the bucket dataset, if possible.',
         argstr='-cbucket %s')
     out_file = File(desc='output statistics file', argstr='-bucket %s')
-    jobs = traits.Int(
+    num_threads = traits.Int(
         desc='run the program with provided number of sub-processes',
         argstr='-jobs %d')
```

Comment:
Might want to set this to `nohash=True`, unless there's reason to think the number of threads affects the output.